src/bin/dist_icu64.mk

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
```

Comment:
Copyright 2016? Is this file a copy of some existing file? If it is, would it be possible to make git show that in the change history?

The way it is now, it looks as if your wrote a new file from scratch, but put the wrong date on it.

Reply:
yes, it's a copy of `dist.mk` from ICU trunk. I'll fix the date.

Reply:
> yes, it's a copy of `dist.mk` from ICU trunk.

Ah, from a file in a different repository? That explains why the diff here shows it as new ... I think it could be valuable if you for those files that are copies listed exact paths of the files they're copies of. That'll make it much easier to track down where things came from if that turns out to be needed in the future.

> I'll fix the date.

It looks as if you forgot to actually do that.

Reply:
> It looks as if you forgot to actually do that.

No, as I said above, I wanted finish using this script (getting the maint tarballs out the door) before coming back to this PR.